webui.lua

```diff
@@ -380,6 +381,18 @@ local function log_line(request, code, length)
     clientip..' - '..user..' ['..time..'] "'..path..'" '..code..' '..length..' "'..referer..'" "'..agent..'"')
 end
 
+local function log_osd(text, duration)
+  if not options.osd_logging then
+    return
+  end
+
+  if duration == nil then
+    duration = 5
+  end
+
+  mp.osd_message(MSG_PREFIX .. text, duration)
+end
```

Comment:
The `duration` argument is never passed to `log_osd()`. For simplicity, I'd prefer:

```suggestion
local function log_osd(text)
  if not options.osd_logging then
    return
  end
  mp.osd_message(MSG_PREFIX .. text, 5)
end
```